src/testing/build/lib/testhelpers/integ.py

```diff
@@ -76,7 +76,7 @@ def assertions(r, endpoint, schemas_path):
     Assert response is successful and validate response body when applicable.
     """
     assert str(r.status_code).startswith("2")
-    if r.headers["Content-Type"].startswith("application/json"):
+    if r.headers["Content-Type"].startswith("application/json") or r.headers["Content-Type"].startswith("text/plain"):
```

Comment:
The failing tests in `orders` and `users` was caused by this file somehow being out of sync with the one in `src/testing/testhelpers`

Reply:
Thanks for fixing this!